amethyst_animation/src/resources.rs

```diff
@@ -134,7 +134,7 @@ where
 /// Defines the hierarchy of nodes that a single animation can control.
 /// Attached to the root entity that an animation can be defined for.
 /// Only required for animations which target more than a single node or entity.
-#[derive(Debug, Clone)]
+#[derive(Debug, Default, Clone)]
```

Comment:
* should use the default in `new`
* should use `derivative` as `derive` only implements it for `T: Default`

Reply:
thx! totally missed that.